tests/src/test/scala/whisk/core/database/LimitsCommandTests.scala

```diff
@@ -55,7 +55,7 @@ class LimitsCommandTests extends FlatSpec with WhiskAdminCliTestBase {
       ns) shouldBe CommandMessages.limitsSuccessfullySet(ns)
 
     val limits = limitsStore.get[LimitEntity](DocInfo(LimitsCommand.limitIdOf(EntityName(ns)))).futureValue
-    limits.limits shouldBe UserLimits(Some(3), Some(7), Some(11), Some(Set("nodejs:6", "blackbox")))
```

Comment:
was/is there some other way to detect this failure (in the test)?

Reply:
Cannot think of. Updated the test now to also include param name in method to make it explicit which value is for what